src/dnd/DraggableController.js

```
@@ -0,0 +1,265 @@
+
+// ---------------------------------------------------------------------------
+// DraggableController implementation
+// ---------------------------------------------------------------------------
+
+class DraggableController extends MiniEventTarget {
+  constructor() {
+    super();
+
+    if (DraggableController._instance) {
+      return DraggableController._instance;
+    }
```

Biome (1.9.4)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor is ignored.

(lint/correctness/noConstructorReturn)